app/renderer/components/navbar_menu.js

```diff
@@ -16,11 +26,24 @@ module.exports = {
         </template>
         <li class="divider"></li>
         <li class="unselectable-text menu-button"><a class="modal-trigger" href="#config-modal">Configuration</a></li>
-        <li class="unselectable-text menu-button"><a class="modal-trigger" href="#about-modal">About</a></li>
+        <li class="unselectable-text menu-button"><a class="modal-trigger" v-on:click="openAbout">About</a></li>
```

Comment:
in this case though you would have only one component with an <a> link and a method to show the popup ?

Reply:
Yes, either a component or a class, It is better to extract anything that is not strictly of the component.
Navbar should contain the button to show the about, but the about itself and its logic is not part of navbar.

Just a question, I don't know if you already tried, but the internal html of a sweetalert could support components? If so, it would make sense to have the about component, and use it within the sweetalert call

Reply:
no it doesn't support components. That would be perfect if it was.

Reply:
However, I see you are using v-on in it

Reply:
maybe i misunderstood. What i am saying is that swal({ html: component }) is not going to work. Is that what you mean also ?

Reply:
No, no, I meant `swal({html: "<about-content></about-content>"})`

This makes a more reasonable thing to have in the method directly

So, in a nutshell, my idea (please, tell me what do you think about this)
1. Wrap swal in a separate class, so we don't have problems with global or compatibility in multiple places
2. Create a component with the content of the about menu and the link logic
3. In navbar, in openMethod, call the wrapped version of swal and simply send an html with the content

Opinions on this?

Reply:
Not sure component work on swal 👎  (sadly) so i guess i have to transfer everything inside a component